alembic_migration/versions/24f8da659c78_add_ratings_to_outings.py

```diff
@@ -0,0 +1,153 @@
+"""Add ratings to outings
+
+Revision ID: 24f8da659c78
+Revises: 8c230a4a0284
+Create Date: 2017-10-13 14:05:52.606170
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '24f8da659c78'
+down_revision = '8c230a4a0284'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        'outings',
```

Comment:
You have to add the columns to the ``outings_archives`` table too. Else they are missing for the archived versions of outings.

Reply:
ok